.github/workflows/main.yml

```diff
@@ -0,0 +1,48 @@
+name: Build, Test, Package and Push
+
+# Controls when the action will run. 
+on: 
+  # Triggers the workflow on pull request events and merges/pushes to main
+  push:
+    branches:
+      - main
+      - release/*
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
think this is the default - maybe not required?

Reply:
yes, those are the default values so that _could_ be dropped

Reply:
Done